examples/example_host.cpp

```diff
@@ -129,6 +129,8 @@ class ExampleHost : public evmc::Host
         (void)topics;
         (void)topics_count;
     }
+
+    evmc_bytes32 get_chain_id() noexcept final { return 0_bytes32; }
```

Comment:
I think this should go into the `tx_context` header.

Reply:
My line of thought was: 1. it's probably not going to be used very often, so why copy it with every `get_tx_context` call; 2. it's not "transaction context" conceptually, but rather more global chain constant.
But I don't insist, can change.